Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -71,10 +71,10 @@ approaches:
 #.  Table and field have a :php:`subtype_value_field` TCA setting
 
     If your table and field have a :php:`subtype_value_field` TCA setting (like
-    :php:`tt_content.list_type` for example) and you want to register a preview
-    renderer that applies only when that value is selected (for example, when a
-    certain plugin type is selected and you can't match it with the "type" of
-    the record alone):
+    :php:`tt_content.list_type` for example) and you want to register a 
+    PreviewRenderer that applies only when that value is selected (for example, 
+    when a certain plugin type is selected and you can't match it with the "type"
```

Comment:
```suggestion
    PreviewRenderer that applies only when that value is selected (assume a certain plugin type is selected and you can't match it with the "type"
```
to reduce the amount of "for example"

Reply:
Sounds good but I did not change the content in those lines, I only replaced the words "preview renderer" with "PreviewRenderer" and needed the rearrange the lines for proper line length...

Reply:
IMHO, either write "preview renderer" or mark it as a class ":php:`PreviewRenderer`".

Reply:
Yeah, might be better. I just orientated myself at the given content. But I did not do it properly -.- Sorry.
"preview renderer" is used throughout the article, just in the `bignums` before it's "PreviewRenderer". So I will undo this change and adapt the only false one. What do you think about it?

Reply:
Let's see ... ?
Anway, thanks for caring!

Reply:
I created a  new commit [143e4a3](https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/2729/commits/143e4a39abba46584afe2efbfedc83abd8c036bd)